test/suites/dev/registar/test_registrar_register.ts

```diff
@@ -52,8 +52,10 @@ describeSuite({
         const containerChainGenesisData = emptyGenesisData();
 
         const tx = polkadotJs.tx.registrar.register(2002, containerChainGenesisData);
-        await polkadotJs.tx.sudo.sudo(tx).signAndSend(alice);
-
+        await tx.signAndSend(alice);
+        await context.createBlock();
+        const tx2 = polkadotJs.tx.registrar.markValidForCollating(2002);
+        await polkadotJs.tx.sudo.sudo(tx2).signAndSend(alice);
         await context.createBlock();
```

Comment:
This can be rewritten into using only one create block like:

```rust
          const { result } = await context.createBlock(
            tx, tx2
          );
```